lib/commands/autocomplete/helper.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs-extra');
+
+const CONSTANTS = require('@src/utils/constants');
+
+module.exports = class Helper {
+    constructor(omelette, commanders = []) {
+        this.commanders = commanders;
+        this.completion = omelette(`ask <command> <subCommand> ${'<option> '.repeat(50).trim()}`);
```

Comment:
omelette does not support unlimited placeholders. Adding up to 50 placeholders for different command options.  Without it only first option would work.